ArduCopter/mode.h

```diff
@@ -151,16 +151,17 @@ class Mode {
     public:
         void start(float alt_cm);
         void stop();
-        void get_climb_rates(float& pilot_climb_rate,
-                             float& takeoff_climb_rate);
+        void get_climb_rates(float& pilot_climb_rate, float& takeoff_climb_rate, float dt);
```

Comment:
Could I suggest just grabbing `dt` from the scheduler - or even calculate your own dt in the function?